src/main/java/io/mosip/mimoto/controller/IssuersController.java

```diff
@@ -33,13 +39,17 @@ public ResponseEntity<Object> getAllIssuers() {
     @GetMapping("/{issuer-id}")
     public ResponseEntity<Object> getIssuerConfig(@PathVariable("issuer-id") String issuerId) {
         ResponseWrapper<Object> responseWrapper = new ResponseWrapper<>();
-        //TODO: Modify id
-        responseWrapper.setId("mosip.inji.properties");
+        responseWrapper.setId(ID);
         responseWrapper.setVersion("v1");
         responseWrapper.setResponsetime(DateUtils.getRequestTimeString());
 
 
-        responseWrapper.setResponse(issuersService.getIssuerConfig(issuerId));
+        IssuerDTO issuerConfig = issuersService.getIssuerConfig(issuerId);
+        responseWrapper.setResponse(issuerConfig);
+        if (issuerConfig == null) {
+            responseWrapper.setErrors(List.of(new ErrorDTO(PlatformErrorMessages.INVALID_ISSUER_ID_EXCEPTION.getCode(), PlatformErrorMessages.INVALID_ISSUER_ID_EXCEPTION.getMessage())));
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(responseWrapper);
+        }
```

Comment:
Let's use http status 404, as 404 for URL requested is invalid or the resource requested, such as an item or a user, does not exist

Reply:
Modified